winml/dll/module.cpp

```diff
@@ -85,6 +89,34 @@ STDAPI DllCanUnloadNow() {
   return S_FALSE;
 }
 
+int32_t WINRT_CALL DllGetExperimentalActivationFactory(void* classId, void** factory) noexcept {
+  try {
+    *factory = nullptr;
+    uint32_t length{};
+    wchar_t const* const buffer = WINRT_WindowsGetStringRawBuffer(classId, &length);
+    std::wstring_view const name{buffer, length};
+
+    auto requal = [](std::wstring_view const& left, std::wstring_view const& right) noexcept {
+      return std::equal(left.rbegin(), left.rend(), right.rbegin(), right.rend());
+    };
+
+    std::wostringstream dummy_class;
+    dummy_class << XSTRINGIFY(WINML_ROOT_NS) << ".AI.MachineLearning.Experimental.Dummy";
```

Comment:
`XSTRINGIFY(WINML_ROOT_NS) ".AI.MachineLearning.Experimental.Dummy"` is an easier way to get the entire string without using streams